brave/src/test/java/com/linecorp/armeria/client/brave/BraveClientIntegrationTest.java

```diff
@@ -153,7 +153,7 @@ public void redirect() throws Exception {
     @Override
     @Test
     public void supportsPortableCustomization() throws Exception {
```

Comment:
you should be able to delete this method override now!

Reply:
After un-overriding, I had to set `RequestContextCurrentTraceContext` to `ITHttp.currentTraceContext` to make
`supportsPortableCustomization()` pass. Should I set it only for
`supportsPortableCustomization()` rather than for all tests?

Reply:
Please put a TODO(adrian) to look into it, that way this doesn't block your change. I can't look into it right now, but will and this is still massive improvement